mangaki/mangaki/templates/base.html

```diff
@@ -168,6 +168,7 @@
 
 <script src="{% url 'js_reverse' %}" type="text/javascript"></script>
 <script src="/static/js/jquery-latest.min.js"></script>
+<script src="https://unpkg.com/vue@2.5.2/dist/vue.min.js"></script>
```

Comment:
We should use a mechanism to enable debug builds rather than min builds in some cases, like my PR (which introduces Vue.js) did.

Reply:
I thought it was inelegant.

Reply:
Well, it depends on your own definition of elegant. But, I, for sure, would like a mechanism to be able to debug Vue.js using Vue.js DevTools and using minimal builds will prevent me from doing it, so I think it is a bad idea, it does not cost anything to have a flag to say: "load the debug build" or "load the min build", we could even rely on `DEBUG` rather than using a `DEBUG_VUEJS` like I did. It's up to you.